storage_bucket/upload_file.py

```diff
@@ -1,14 +1,16 @@
-from attr import dataclass
-from typing_extensions import final
+from storage_bucket.bucket import get_bucket
 
-from storage_bucket.get import GetBucket
 
-
-@final
-@dataclass(frozen=True, slots=True)
-class UploadFile(object):
+def upload_file(
+    *,
+    file_content: bytes,
+    storage_bucket_name: str,
+    filename: str,
+    content_type: str = 'application/octet-stream',
+    **kwargs: dict,
+) -> None:
     """
-    Uploades contents in file_data to a google cloud storage bucket.
+    Upload contents in file_data to a google cloud storage bucket.
```

Comment:
Maybe it will be better: `Upload content of file_data to a google cloud storage bucket.`?